public/js/monetization-api-buttons.js

```diff
@@ -55,6 +56,38 @@ function renderReaderIdForm(selector) {
   document.querySelector(selector).appendChild(form);
 }
 
+/**
+ * renderPublicationIdForm
+ * Creates a button to set a PublicationId
+ * @param {string} selector
+ */
+function renderPublicationIdForm(selector) {
```

Comment:
Given the similarity with `renderReaderIdForm`, might be worth making a generic function that can be used by both?

Reply:
Yeah, I was thinking a similar thing, but felt like this was a quick win. I can do a follow-up that moves this to a utility that can be shared across content pages.